docs/docs/mlflow-3/index.mdx

```diff
@@ -0,0 +1,116 @@
+---
+sidebar_label: MLflow 3.0
+sidebar_position: 1
+---
+
+import Tabs from "@theme/Tabs";
+import TabItem from "@theme/TabItem";
+import { APILink } from "@site/src/components/APILink";
+import { CardGroup, PageCard } from "@site/src/components/Card";
+
+# MLflow 3.0
+
+Discover the next generation of MLflow, designed to streamline your AI experimentation and accelerate your journey from idea to production. MLflow 3.0 brings cutting-edge support for GenAI workflows, enabling seamless integration of generative AI models into your projects.
+
+## What is MLflow 3.0?
+
+MLflow 3.0 delivers best-in-class experiment tracking, observability, and performance evaluation for machine learning models, AI applications, and generative AI agents!
+MLflow 3.0 introduces a variety of exciting improvements, including:
+- Centrally track and analyze the performance of your models and generative AI applications **across all environments**, from interactive queries in a development notebook through production batch or real-time serving deployments.
+- Gain insights into model performance with **comprehensive metrics and lineage tracking**, enabling better decision-making for production readiness.
+- Streamline the handling of model artifacts and evaluation results with a **cohesive structure** that improves observability and traceability.
+
+Model Tracking helps you create and evaluate different model/agent configurations in your experiments, <APILink fn="mlflow.entities.LoggedModel">LoggedModel</APILink> helps you find the best ones for production usage, and tracing helps enhance observability in your model or GenAI applications.
+
+
+## Model Tracking with LoggedModel
+
+In MLflow 3.0, we introduce a refined architecture along with revamped APIs and UI, tailored to enhance deep learning and generative AI workflows. 
```

Comment:
@serena-ruan Can we switch the order of deep learning and generative AI throughout the sentences in these docs? The emphasis of MLflow 3.0 should be GenAI.

Can we also switch the order of the deep learning and GenAI sub-tabs?